### DISCUSSION TITLE
initiatives listing layout

### DISCUSSION THREAD

**Discussion by ahmedabbous** [TDRA-ae/aegov-dls#21] [General]
I'm examining the initiatives page on the FAHR website (https://www.fahr.gov.ae/Portal/en/about-fahr/initiatives.aspx) and noticed that it features a series of images as initiatives. while the design for initiatives in the DLS2.0 Figma project is significantly differs from the current content presentation on the FAHR website
please advice what is the most effective layout for presenting the initiatives on the listing page, whether it's more appropriate to adapt the website content to align with the Figma design or if you suggest other option (like media design) to accommodate the existing content style and structure on the website?
please note the details for each initiatives is different from one to one, each initiative has different style

**Comment by moiz-conceptlz** [comment]
Hi @ahmedabbous 

Content is a key factor on any website. Creating content that helps a user, and guides them to better understand details is the reason we make pages on websites.

A series of images may not be the best approach. However, following a layout similar to perhaps : https://mohap.gov.ae/en/about-us/projects-and-initiatives would be the most suitable approach. Having cards that display initiatives, and creating a page to explain the initiative would be the most user friendly approach.

If the initiative has a separate website already, allow a link to that website as well.
